Adding the Media Type column to registry (#34)
core-wg/cf-reg-update: draft-ietf-core-cf-reg-update.md

A new column with the title "Note" has been added to the registry, which contains information about expected checks.

## Adding the Media Type Column to the Registry

> To make it easier for users of the CoAP Content-Formats registry to locate details about the media type used by a CoAP Content-Format, and to enforce the existence of the media type prior to the registration of a new entry, IANA is requested to add a new column "Media Type" to the registry.

Reviewer: ```suggestion
To assist users of the CoAP Content-Formats registry in finding detailed information about the media type associated with each CoAP Content-Format, and to ensure that a media type exists before a new entry can be registered, IANA is requested to add a new column "Media Type" to the registry.```

> This new column can be placed directly to the right of the current "Content Type" column.

Reviewer: ```suggestion
This new column can be placed directly to the right of the existing "Content Type" column.
```

The "Media Type" field for each entry lists the (base) media type name and provides a hyperlink to registration information for that media type as recorded by IANA.

> In case the media type is provisional, the hyperlink can point directly to the IANA Provisional Standard Media Type registry.

Reviewer: Done in [3dedc9b](https://github.com/core-wg/cf-reg-update/pull/34/commits/3dedc9b8209b7bdb6a2ce5dc3f4bd177ffefb915)

A new column with the title "Note" has been added to the registry, which contains information about expected checks.

## Adding the Media Type Column to the Registry

> To assist users of the CoAP Content-Formats registry in finding detailed information about the media type associated with each CoAP Content-Format, and to ensure that a media type exists before a new entry can be registered, IANA is requested to add a new column "Media Type" to the registry.```

Reviewer: ```suggestion
To assist users of the CoAP Content-Formats registry in finding detailed information about the media type associated with each CoAP Content-Format, and to ensure that a media type exists before a new entry can be registered, IANA is requested to add a new column "Media Type" to the registry.
```
